### DISCUSSION TITLE
Update Version 1 of 1.12.0

### DISCUSSION THREAD

**Discussion by PaxJaromeMalues** [EwyBoy/OreTweaker#85] [Ideas]
1.12.2 is still a VERY broadly used MC Version.
It would be awesome, if your mod release for 1.12 could be updated from OreTweaker v1 to OreTweaker versions 2 or 3, if tho probably with limited functionality as v2 and v3 probably need some of the higher MC version features and hooks to work properly.

**Comment by EwyBoy** [comment]
Its just to much work, right now I am spending all my RnD time on making Ore Tweaker work for 1.19 plus.
